go/cmd/dolt/commands/sql.go

```diff
@@ -676,6 +654,15 @@ func runShell(ctx *sql.Context, se *sqlEngine, mrEnv env.MultiRepoEnv) error {
 			}
 		}
 
+		if err == nil {
+			returnedVerr = writeRoots(ctx, se, mrEnv, initialRoots)
+
+			if returnedVerr != nil {
+				shell.Println(returnedVerr.Verbose())
```

Comment:
Sure we aren't going to double-print this error? Check and see how it's handled at the call site.

Reply:
Good catch, it would eventually get double printed.